spec/index.html

```diff
@@ -732,12 +754,15 @@ <h2>Literals</h2>
         not defined by this specification.</li>
     </ol>
 
-    <p><dfn data-local-lt="term-equal">Literal term equality</dfn>: Two literals are term-equal (the same
-      RDF literal) if and only if the two <a>lexical forms</a>,
-      the two <a>datatype IRIs</a>, and the two
-      <a>language tags</a> (if any) compare equal,
-      character by character. Thus, two literals can have the same value
-      without being the same RDF term. For example:</p>
+    <p><dfn data-local-lt="term-equal">Literal term equality</dfn>:
+      Two literals are term-equal (the same <a>RDF literal</a>)
+      if and only if the two <a>lexical forms</a>,
+      the two <a>datatype IRIs</a> compare equal, character by character,
+      the two <a>language tags</a> (if any) compare equal, as if normalized to lower case, and
```

Comment:
```suggestion
      the two <a>language tags</a> (if any) compare equal after being normalized to lower case, and
```

Reply:
The spec does not requiring normalizing to lower case, only for comparison purpose. Your suggested change would seem to suggest that they are normalized to lower case. It's only the comparison which is case insensitive.